ramls/subscribers/fee-fine-with-loan-closed.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Should this be renamed now that the event has been renamed?

Reply:
Thanks, renamed.